packages/terra-collapse/tests/jest/Collapse.test.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import IconCaretRight from 'terra-icon/lib/icon/IconCaretRight';
+import Collapse from '../../src/Collapse';
+
```

Comment:
The icon prop is required but most of these tests are not providing an icon element.